src/main/resources/data/order-template-categories/amazon_book_orders.json

```diff
@@ -0,0 +1,4 @@
+{
+  "id": "4dee318b-f5b3-40dc-be93-cc89b8c45b6f",
+  "name": "Fiction"
```

Comment:
I remember asked on refinement that we do not need sample data for Template Categories

Reply:
it was being used by tests only. I will move to test section in this case